src/Components/NavBar/NavBarMobileMenu/NavBarMobileMenuAuthentication.tsx

```diff
@@ -84,6 +84,8 @@ export const NavBarMobileMenuLoggedIn: React.FC<NavBarMobileMenuLoggedInProps> =
 
   return (
     <>
+      <NavBarMobileSubMenu menu={menu}>{menu.title}</NavBarMobileSubMenu>
```

Comment:
**question**: I think we should also move "Activity" under "Account" ([source](https://www.figma.com/file/SlzmAJV6zqJDmAO8GoSjiC/Activity-%2F-Save-%2F-Follow-%2F-Create?node-id=1504%3A77970)) 

<img width="630" alt="for-george" src="https://user-images.githubusercontent.com/3513494/188619103-e57ceff5-d549-4e8b-a01e-cd27cb3e9c4e.png">

Reply:
done

Reply:
thanks 🙏